src/test/neorados/pool.cc

```diff
@@ -81,8 +81,9 @@ class NeoRadosPool : public CoroTest {
     co_return co_await lookup_pool(pname);
   }
 
-  auto delete_pool(std::string pname) {
-    return rados().delete_pool(pname, asio::use_awaitable);
+  auto delete_pool(std::string pname)-> boost::asio::awaitable<void> {
```

Comment:
`boost::asio::awaitable<void> delete_pool(std::string pname) {`

Reply:
The second override of delete_pool should probably match the new return type:
`auto delete_pool(std::int64_t pid)`

Reply:
It is not required because we are not calling second delete_pool. I will remove it.